prisma/schema.prisma

```diff
@@ -206,75 +207,102 @@ model Comment {
-  createdAt   DateTime  @default(now())
-  question    Question  @relation(fields: [questionId], references: [id])
-  questionId  Int
-  author      User      @relation(fields: [authorId], references: [id])
-  authorId    String
-  votes       Vote[]
-  upvotes     Int       @default(0)
-  downvotes   Int       @default(0)
-  totalanswers  Int     @default(0)
-  parentId    Int?      
-  responses   Answer[]  @relation("AnswerToAnswer")
-  parent      Answer?   @relation("AnswerToAnswer", fields: [parentId], references: [id])
-  updatedAt   DateTime  @updatedAt
+  id           Int      @id @default(autoincrement())
+  content      String
+  createdAt    DateTime @default(now())
+  question     Question @relation(fields: [questionId], references: [id])
+  questionId   Int
+  author       User     @relation(fields: [authorId], references: [id])
+  authorId     String
+  votes        Vote[]
+  upvotes      Int      @default(0)
+  downvotes    Int      @default(0)
+  totalanswers Int      @default(0)
+  parentId     Int?
+  responses    Answer[] @relation("AnswerToAnswer")
+  parent       Answer?  @relation("AnswerToAnswer", fields: [parentId], references: [id])
+  updatedAt    DateTime @updatedAt
 
   @@index([questionId])
   @@index([authorId])
   @@index([parentId])
 }
 
-
 model Vote {
-  id        Int      @id @default(autoincrement())
-  questionId Int?    
-  question  Question? @relation(fields: [questionId], references: [id])
-  answerId  Int?     
-  answer    Answer?  @relation(fields: [answerId], references: [id])
-  commentId Int?     
-  comment   Comment? @relation(fields: [commentId], references: [id])
-  userId    String
-  user      User     @relation(fields: [userId], references: [id])
-  voteType  VoteType 
-  createdAt DateTime @default(now())
+  id         Int       @id @default(autoincrement())
+  questionId Int?
+  question   Question? @relation(fields: [questionId], references: [id])
+  answerId   Int?
+  answer     Answer?   @relation(fields: [answerId], references: [id])
+  commentId  Int?
+  comment    Comment?  @relation(fields: [commentId], references: [id])
+  userId     String
+  user       User      @relation(fields: [userId], references: [id])
+  voteType   VoteType
+  createdAt  DateTime  @default(now())
 
-  @@unique([questionId, userId])  
-  @@unique([answerId, userId])    
-  @@unique([commentId, userId])  
+  @@unique([questionId, userId])
+  @@unique([answerId, userId])
+  @@unique([commentId, userId])
 }
 
 enum VoteType {
   UPVOTE
   DOWNVOTE
 }
+
 enum PostType {
   QUESTION
   ANSWER
 }
+
 enum CommentType {
   INTRO
   DEFAULT
 }
+
+model GithubUser {
+  id         Int          @id @default(autoincrement())
+  userId     String       @unique
+  username   String       @unique
+  email      String?      @unique
+  publicName String?
+  isLinked   Boolean      @default(false)
+  user       User?        @relation(fields: [userId], references: [id])
+  bountyInfo BountyInfo[]
+}
+
+model BountyInfo {
```

Comment:
This is bad since the single source of truth should be the notion database from `bounty-hook`

Reply:
Initially thought the same but found some flaws like
1. It would be difficult for querying from doc than database
2. GithubUser ,BountyInfo and User table all are connected as relational database through single userId for ease in querying 
3. The information is saved both in notion doc (in bounty-hook) and in db. (in case of data loss or data manipulation by error, as seen in bookmark, data in the notion doc would be safe)

Still ready to change if needed